### PR TITLE
Plugins for baidu analytics

### PR DESCRIPTION
**Pull request by LeoooY**
## Motivation

I want my docusaurus site to support baidu analytics for tracking.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

yes.

## Test Plan
All I need is to insert a script tag to the final index.html file with specific trackingID.

## Related PRs
none

### CODE REVIEW

**Comment by slorber** [comment]
Thanks @LeoooY 

After discussing this internally, we think this plugin is not a good fit for our repo. We only include plugins that we use on our own website and know how to maintain.

You'd better publish the plugin under your own name, and add it to our community plugins doc.

Maybe we'll create a docusaurus org to host more community plugins, or create a docusaurus-community monorepo with most popular contribs, but for now no decision has been made regarding this, and we encourage you to maintain your own code over time

File: website/docs/using-plugins.md [hunk 1]
```diff
@@ -512,6 +512,31 @@ module.exports = {
...
+npm install --save @docusaurus/plugin-baidu-analytics
+```
+
+**Configuration**
+
+```js title="docusaurus.config.js"
+module.exports = {
+  plugins: ['@docusaurus/plugin-baidu-analytics'],
+  themeConfig: {
+    gtag: {
```
Comment (slorber) [inline website/docs/using-plugins.md:531]: that does not look good to have gtag here

Comment (LeoooY) [reply website/docs/using-plugins.md:531]: my mistake, it should be like this.
```
module.exports = {
  plugins: [
      [ '@docusaurus/plugin-baidu-analytics',{ 
        trackingID: 'xxxxxxxxx',
        // Optional fields.
        trackingDEV: true, // Should insert tracking script in development? default to false} 
      ]
     ],
}
```
and thanks for your reply, I`d like to publish the plugin under my own name, and add it to community plugins doc.

Comment (slorber) [reply website/docs/using-plugins.md:531]: great thanks

this triggered a discussion about a potential docusaurus org, will get back to you if a decision is made

**Pull request closed without merging**